### PR TITLE
Add Rennes

### PR DESCRIPTION
**Pull request by XavCC**
- In locations.json
- Places in rennes.json

### CODE REVIEW

File: locations/locations.json [hunk 1]
```diff
@@ -145,6 +145,12 @@
         "lon" : -1.2557418,
         "defaultZoom": "15"
     },
+    "rennes" : {
+        "name" : "Rennes"
```
Comment (avernois) [inline locations/locations.json]: There is a missing comma at the end of the line.

**Comment by avernois** [comment]
Hello,

\o/ we're almost good :)
There is a missing comma in the locations.json (line 149).

**Comment by XavCC** [comment]
Thanks a lot 
Fixed
@avernois

**Comment by avernois** [comment]
Merged \o/

Thanks!

**Comment by avernois** [comment]
note: https certificate generation is not automatic, so it'll be only available over http://rennes.devfriendlyplaces.net until I launch the script (this evening if I do not forget :)

**Pull request merged**